src/main/java/ch/uzh/ifi/seal/soprafs20/entity/Game.java

```diff
@@ -51,25 +57,49 @@
     @ElementCollection
     private List<String> clues = new ArrayList<String>();
 
-    @Column(nullable = false)
-    private int timestamp;
+    @Column
+    @ElementCollection
+    private List<Integer> score = new ArrayList<>();
```

Comment:
This is needed because after the end of the game we need to add all the points accumulated in each round to get the points collected during whole game.